packages/atomic/src/pages/examples/custom.html

```diff
@@ -60,13 +60,20 @@
       import {resultContext} from '/build/index.esm.js';
 
       class CustomTemplateComponent extends HTMLElement {
+        initialized = false;
         connectedCallback() {
```

Comment:
I'm wondering if perhaps we should add a `disconnectedCallback` to clear listeners. This would probably be better demonstrated in the non-template custom component.